### PR TITLE
Adding Google Tag Manager

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: demo/src/index.html [hunk 1]
```diff
@@ -17,10 +17,18 @@
   <link rel="stylesheet" href="fonts/fonts.css">
   <link rel="stylesheet" href="css/vendor.css">
   <link rel="stylesheet" href="css/all.css">
+
+  <!-- Google Tag Manager -->
```
Comment (de-code) [inline demo/src/index.html:21]: minor: based on the [quickstart documentation](https://developers.google.com/tag-manager/quickstart) this should be:

> as close to the opening `<head>` tag as possible

I guess in this case the loading of the CSS and fonts could affect the accurate measurement within Google Tag Manager. (Although we are probably not going to try to optimize it that much, but are more interested in whether the site is used)

**Review by de-code — APPROVED** [review]

File: demo/src/index.html (hunk 1 above)
Comment (discodavey) [reply demo/src/index.html:21]: It does not make any difference where it is in the head. This is Google being Google.

Comment (de-code) [reply demo/src/index.html:21]: Why do you think it does not make any difference? Do you not believe that the location of the script tag influences when it is being loaded by the browser?

The following StackOverflow answer seem to give a reasonable explanation: https://stackoverflow.com/a/46065442/8676953

Comment (discodavey) [reply demo/src/index.html:21]: Just reading the article I can see what they are saying but there is not much to load on the page as it is. Over the years of working with Analytics and other developers it was agreed that the difference in load times is not going make any gains that will be an advantage there has not been any difference in tracking in putting it in either place.

We have the code in the same place for Journal which is another reason why I placed it where I did so it was consistent.

Comment (de-code) [reply demo/src/index.html:21]: Okay, it would definitely be more relevant to Journal.

**Pull request merged**